package-lock.json

```diff
@@ -1,10436 +0,0 @@
-{
```

Comment:
Rather than deleting the file, it seems necessary to make it the initial state so that there are no changes.

Reply:
Well.. I am not familiar with how to use git...  
is this right?